README.md

```diff
@@ -1,24 +1,314 @@
+pages in the same website will be unaffected.
+
+If we want our CSS to carry across various pages, we can use an external
+stylesheet. This is a separate CSS file that we link in the `head` of HTML
+documents.
+
+```
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+
+  <p>This is a paragraph.</p>
+
+  </body>
+</html>
+```
+
+With the `link` tag, we can use the relation attribute to define the target as a
+stylesheet and the link source our CSS file that contains all the CSS we want to
+use on our site. This is by far the easiest way to link CSS to HTML and apply
+styles across all of our pages.
+
+## Utilize Various Types of CSS Selectors
+
+CSS gives us a wide range of ways that we can select elements on the page. Some
+of these you will use more than others, but it's a good idea to know them all so
+you can choose the right one when you need it.
+
+#### ID and Class Selectors
+
+ID selectors target all elements with a specific ID attribute value. The way we
+specify an ID selector in a CSS rule is is to follow the element name with a
+hash tag and then the ID attribute value we want to match.
+
+```
+p#introduction {
+  color: blue;
+}
+```
+
+In this case, the browser will look for a `p` element with the ID attribute
+"introduction" and apply the CSS to that element. ID selectors are useful when
+you want to give a single element on the page an unique identity and set it
+apart from everything else.
+
+Class selectors target all elements with a class attribute value matching the
+selector name. We specify a class selector using the period symbol followed by
+the name value.
+
+```
+p.alert {
+  color: red;
+}
+```
+
+The difference between IDs and classes is that IDs are meant for one element on
+the page that has unique identity where class selectors are meant to be spread
+throughout the page across multiple elements.
+
+#### Compound Selectors
+
+Compound selectors let us apply the same CSS rules to multiple elements at once.
+If we want to make both `h1` and `h2` elements display green, we use both as
+selectors, separated with a comma.
+
+```
+h1, h2 {
+  color: green;
+}
+```
+
+This eliminates the need to rewrite a new CSS rule containing the same styles
+for different elements.
+
+#### Descendant Selectors
+
+Descendant selectors target elements that are descendants of the matching
+selector name. A descendant selector is indicated by a space in between one
+selector and another selector.
+
+```
+article p {
+  color: blue;
+}
+```
+
+In this case, only `p` elements within the `header` element will recieve the
+styling.
+
+#### Child Selectors
+
+The child selector targets all elements that are the immediate children of a
+specified element.
+
+```
+article > p {
+  color: blue;
+}
+```
+
+Only `p` tags one level down from `article` will display as blue. If there are
+`p` tags within an `aside` element within the `article` element, they will not
+recieve the same instructions.
+
```

Comment:
Maybe say like "Here's where these selectors start getting pretty advanced."

Find some line where here's what you use all de damn day versus these kinda less popular selectors (in my experience anyway).